UnityProject/Assets/Scripts/Objects/Drawers/Morgue.cs

```diff
@@ -131,9 +131,8 @@ private void UseEmag(HandApply interaction)
 			alarmBroken = true;
 
 			Chat.AddActionMsgToChat(interaction,
-					$"You wave the {interaction.HandObject.name.ToLower()} over the {name.ToLower()}'s electrical panel. " +
 					"The status panel flickers and the buzzer makes sickly popping noises. You can smell smoke...",
```

Comment:
Looks like the Morgue hasn't been updated to use up a charge (or display the removed text here).
You'll want get the emag component and check that it has charges on line 53 via `TryGetComponent<>(), then you can pass this variable into the `Morgue.UseEmag()` method, here, to call the `UseCharge()` method on the component.

Reply:
alright, cool. Done.